src/core/entities/MRMediaEntity.js

```diff
@@ -117,7 +117,7 @@ export class MRMediaEntity extends MRDivEntity {
      * @function
      * @description Callback function of MREntity - handles setting up this media and associated 3D geometry style (from css) once it is connected to run as an entity component.
```

Comment:
can you add `(async)` to the description like you did for the other newly async functions?